lua/nvim-tree/diagnostics.lua

```diff
@@ -17,7 +17,7 @@ local COC_SEVERITY_LEVELS = {
 }
 
 ---Absolute Node path to LSP severity level
----@alias NodeSeverities table<string, lsp.DiagnosticSeverity>
+---@alias NodeSeverities table<string, vim.diagnostic.Severity>
```

Comment:
`lsp.DiagnosticSeverity` and `vim.diagnostic.Severity` are effectively the same. It's not clear which is preferred, so I went with the one from `vim.Diagnostic`